### ISSUE TITLE
PHPMyAdmin Error: Could Not Save Configuration - Duplicate entry ' ' for key 'PRIMARY'

### ISSUE THREAD

**Issue by pllealfunes** [susanBuck/e2-fall20#99]
I started watching this week's videos and I started fixing my .env file as I didn't have `APP_NAME` as the name of my app which is “Mystery Animal” and didn't have `DB_NAME` as p3 but decided to keep it as is . I had it like this:

> APP_NAME='My App'
APP_URL='http://e2p3.loc'
DB_HOST='127.0.0.1'
DB_NAME='mysteryanimal'
DB_USERNAME='root'
>DB_PASSWORD=''

I then went on to make some css edits and some in my `GameController`. I decided to do a `php console App fresh` and when I refreshed phpmyadmin I got an error saying `Could Not Save Configuration - Duplicate entry ' ' for key 'PRIMARY'`. I cleared my browser history and cache, and changed back my edits but the error still happens. I don't see any duplicates in my table.

https://github.com/pllealfunes/e2/tree/master/p3

<img width="436" alt="Screen Shot 2020-12-03 at 7 58 20 PM" src="https://user-images.githubusercontent.com/51715816/101110404-e81d2700-35a6-11eb-832d-2b2a25ead596.png">

<img width="1440" alt="Screen Shot 2020-12-03 at 7 58 28 PM" src="https://user-images.githubusercontent.com/51715816/101110327-bf952d00-35a6-11eb-98ce-48aec47b617d.png">

**Comment by maqboolk** [comment]
@pllealfunes  
I haven't watched this week video yet, but seems like you trying to insert a record with same primary key. 
Try re-building your table again, that way you will have clean slate and primary key will start at 1.

Is your primary key set to Auto increment?

Note: you will loose all your data.

**Comment by maqboolk** [comment]
@pllealfunes 

Did you omit ID while seeding data? did you unset the ID while seeding the data?

**Comment by pllealfunes** [comment]
@maqboolk I rebuilt my table and added my seeds but I still get the error on refresh. I even deleted the database and played my game first, not adding the seeds yet, and still get the error. I did leave out ID in my seeded data.

Seed data:
>$this->app->db()->createTable('results', [
            'guess' => 'varchar(255)',
            'outcome' => 'tinyint(1)',
       > ]); 

<img width="1440" alt="Screen Shot 2020-12-03 at 9 38 44 PM" src="https://user-images.githubusercontent.com/51715816/101114880-60d4b100-35b0-11eb-8171-632e27a4ab54.png">

**Comment by maqboolk** [comment]
@pllealfunes 
Try creating table manually instead of using command. 


Try restarting server before creating table manually.

**Comment by susanBuck** [comment]
@pllealfunes 

Want to make sure I'm understanding correctly -

This error happens every time you refresh phpMyAdmin?

Cloning your codebase now to see if I can recreate.

**Comment by pllealfunes** [comment]
@maqboolk  

When you mean manually, do you mean creating the tables in phpmyadmin? I had done that, sorry I wasn't clear about that. I had also restarted XAMPP and nothing. I decided to just restart my computer and now it works. I also just added the code to unset the ID and it still works. Not sure what was wrong but will keep an eye an eye on it. Thank you for the help :)

**Comment by maqboolk** [comment]
> @maqboolk
> 
> When you mean manually, do you mean creating the tables in phpmyadmin? I had done that, sorry I wasn't clear about that. I had also restarted XAMPP and nothing. I decided to just restart my computer and now it works. I also just added the code to unset the ID and it still works. Not sure what was wrong but will keep an eye an eye on it. Thank you for the help :)

Yes, Creating table using phpmyadmin.  Maybe it was some cache issue. i am glad its working now.

**Comment by pllealfunes** [comment]
@susanBuck  Yes, I kept getting the error every time I refreshed phpMyAdmin. I had made some edits (as stated above) and when I refreshed phpMyAdmin I would get the error. I started to change back the edits one by one to see what could have been the issue but I would still get the error and I even recreated the table several times. It is working now having restarted my computer, so maybe it was a cache issue like @maqboolk suggested.
